src/core/Akka.API.Tests/CoreAPISpec.ApproveClusterTools.approved.txt

```diff
@@ -373,20 +375,21 @@ namespace Akka.Cluster.Tools.Singleton
         public static Akka.Cluster.Tools.Singleton.ClusterSingletonProxySettings Create(Akka.Configuration.Config config) { }
         public Akka.Cluster.Tools.Singleton.ClusterSingletonProxySettings WithBufferSize(int bufferSize) { }
         public Akka.Cluster.Tools.Singleton.ClusterSingletonProxySettings WithRole(string role) { }
-        public Akka.Cluster.Tools.Singleton.ClusterSingletonProxySettings WithSingletonIdentificationInterval(string singletonIdentificationInterval) { }
+        public Akka.Cluster.Tools.Singleton.ClusterSingletonProxySettings WithSingletonIdentificationInterval(System.TimeSpan singletonIdentificationInterval) { }
         public Akka.Cluster.Tools.Singleton.ClusterSingletonProxySettings WithSingletonName(string singletonName) { }
     }
     public enum ClusterSingletonState
```

Comment:
If I recall correctly, none of these values are transmitted over the wire - correct?

Reply:
yes, don't think it's send over the wire.
It was more like a bug there, because the string value was not used for anything there. Instead it was using the value, which was already there 

```
return Copy(singletonIdentificationInterval: SingletonIdentificationInterval);
```
vs
```
return Copy(singletonIdentificationInterval: singletonIdentificationInterval);
```
'S' vs 's'

can also add a "dummy" overload for that if needed

Reply:
Man that's a nasty bug. Yeah, add an overload for good measure there.